src/main/java/com/trzewik/blackjack/Croupier.java

```diff
@@ -0,0 +1,12 @@
+package com.trzewik.blackjack;
+
+public class Croupier extends Contestant {
+
+    int casino;
+
+    public Croupier(){
+        this.name = "Croupier";
```

Comment:
to nie jest this...

Reply:
this odnosi sie do tego obiektu, czyli do Croupiera w tym wypadku. Ty sie odnosisz do jego super klasy! czyli uzywasz slowka super.doSth().